doc/architecture/sensor_data.adoc

```diff
@@ -1,9 +1,6 @@
 = Sensor data
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu. Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a. Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+Sensor data imitates the output of real sensors.
+It is generated from ground-truth data, feature data or from sensor data.
+All information regarding the environment is given with respect to the virtual sensor coordinate system, except for feature data, which is given with respect to the physical sensor coordinate system.
```

Comment:
Aside from feature data, information regarding the environment is given with respect to the virtual sensor coordinate system, which is defined....
Feat data is given with respect to the physical sensor coordinate system, which is defined by....